insights/parsers/ls_etc.py

```diff
@@ -29,12 +31,12 @@
     ... lrwxrwxrwx.  1 0 0   15 Jul  6 23:32 S97rhnsd -> ../init.d/rhnsd
     ... '''
     >>> ls_etc = LsEtc(context_wrap(LS_ETC))
```

Comment:
I thought the whole point of the new documentation examples was that we weren't importing the parser or `context_wrap` and creating the parser arbitrarily, we were showing that (in this case) the `ls_etc` parser object exists by a type check, e.g.:
```python
>>> type(ls_etc)
<class 'insights.parsers.ls_etc.LsEtc'>
```
When using `doctest.testmod` in the tests, this would then be simulated by doing the `context_wrap` and parser object creation in a global environment for the test module.  Look for `testmod` in other parser tests for examples of how this can be done.